### ISSUE TITLE
Warning generated in 0.5: symbol --> Symbol

### ISSUE THREAD

**Issue by scheinerman** [simonster/Memoize.jl#10]
When using this module I get the following warning:
```
julia> using Memoize

julia> @memoize f(x) = x+1
WARNING: symbol is deprecated, use Symbol instead.
 in depwarn(::String, ::Symbol) at ./deprecated.jl:64
 in symbol(::String, ::Vararg{String,N}) at ./deprecated.jl:30
 in @memoize(::Expr, ::Vararg{Expr,N}) at /Users/ers/.julia/v0.5/Memoize/src/Memoize.jl:14
 in eval(::Module, ::Any) at ./boot.jl:234
 in eval(::Module, ::Any) at /Applications/Julia-0.5.app/Contents/Resources/julia/lib/julia/sys.dylib:?
 in eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:64
 in macro expansion at ./REPL.jl:95 [inlined]
 in (::Base.REPL.##3#4{Base.REPL.REPLBackend})() at ./event.jl:68
while loading no file, in expression starting on line 0
(::#28#f) (generic function with 1 method)
```

A quick scan of the source code reveals just two instances of `symbol`; just change those to `Symbol`?

**Comment by simonster** [comment]
This is already fixed on master. Unfortunately the tests still seem to be broken on 0.5 and I haven't had time to get to this.

**Comment by cossio** [comment]
So, this should get fixed in the next release? 0.0.2?
thanks
